other/block-pod-exec-by-namespace.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: deny-exec-by-namespace-name
+  annotations:
+    policies.kyverno.io/title: Block Pod Exec by Namespace Name
+    policies.kyverno.io/category: Sample
+    policies.kyverno.io/minversion: 1.4.2
+    policies.kyverno.io/subject: Pod
+    policies.kyverno.io/description: >-
+      The `exec` command may be used to gain shell access to a given Pod. While this can
```

Comment:
`The "exec" command may be used to gain shell access, or run other commands, in a Pod's container.`